Cargo.toml

```diff
@@ -23,4 +23,4 @@ brotli = "1.1.2"
 flate2 = "0.2"
 rand = "0.4"
 thrift = "0.0.4"
-x86intrin = "0.4.3"
+x86intrin = "0.4.3"
```

Comment:
Looks like there is no new line at the end of file.

Reply:
Yes, I'll revert this change.